libs/SalesforceNetwork/SalesforceNetwork/SalesforceNetwork/Classes/Network/Action/CSFAction.m

```diff
@@ -354,9 +354,9 @@ - (void)start {
         return;
     }
     
-    [self willChangeValueForKey:@"executing"];
```

Comment:
I think the properties are executing and finished and the KVO key paths are isExecuting and isFinished (see the next paragraph in the doc). Also with the executing key path, my test below does not succeed.